### PR TITLE
fix #36075 sinarharian.com.my

### LINKED ISSUE #36075: sinarharian.com.my
### Issue URL (Incorrect Blocking)
[https://www.sinarharian.com.my](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.sinarharian.com.my)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 3.0.0
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | Default
Filters: | AdGuard Mobile Ads,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection
Other extensions used: | NIL

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/36075

### CODE REVIEW

File: MobileFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -23,6 +23,9 @@
 @@||kocaelibarisgazetesi.com/images/banner/l3oo.jpg
 ! https://forum.adguard.com/index.php?threads/www-dailymotion-com.31223
 @@||dmcdn.net^$app=com.dailymotion.dailymotion
+! https://github.com/AdguardTeam/AdguardFilters/issues/36075
+@@||securepubads.g.doubleclick.net^$domain=sinarharian.com.my
```
Comment (Alex-302) [inline MobileFilter/sections/whitelist.txt]: too wide rule

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -31,6 +31,7 @@ propakistani.pk##.pp-single-social-share
 aorus.com##ul.social
 di.fm##.social-region
 zirochka.ua##.share-btns__list
+sinarharian.com.my##.articleDetails > .bgwhite > .icon-bar
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt]: why not just `a[class*="addthis"]`?

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by tvinzz** [comment]
made rules more exact and changed social rule for sharing

File: MobileFilter/sections/whitelist.txt [hunk 3]
```diff
@@ -23,6 +23,10 @@
 @@||kocaelibarisgazetesi.com/images/banner/l3oo.jpg
 ! https://forum.adguard.com/index.php?threads/www-dailymotion-com.31223
 @@||dmcdn.net^$app=com.dailymotion.dailymotion
+! https://github.com/AdguardTeam/AdguardFilters/issues/36075
+@@||securepubads.g.doubleclick.net/tag/js/gpt.js^$domain=sinarharian.com.my
+@@||securepubads.g.doubleclick.net/gpt/pubads_impl_2019070801.js^$domain=sinarharian.com.my
```
Comment (Alex-302) [inline MobileFilter/sections/whitelist.txt]: `pubads_impl_2019070801.js` number is not constant. Replace by `/pubads_impl_*.js`
`^` is unnecessary

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: MobileFilter/sections/whitelist.txt (hunk 3 above)
Comment (tvinzz) [reply MobileFilter/sections/whitelist.txt]: @Alex-302, replaced by `/pubads_impl_*.js` and removed the `^`

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**